remc2/portability/port_net.cpp

```diff
@@ -735,7 +735,7 @@ namespace MyNetworkLib {
 
 	void NetworkClass::SendGiveIPMessage(asio::ip::udp::endpoint sender, int port) {		
 		shadow_myNCB nullNCB;
-		nullNCB.ncb_command_0 = 999;
+		nullNCB.ncb_command_0 = 254;
 
```

Comment:
shouldn't 255 be the max here? I though I actually changed this in one of my sonar fixes of your last PR

Reply:
255 - 0xff is sometimes used by the original code, so I want to avoid this value so that it doesn't cause a problem anywhere. True, I think 0xff does use it for a different variable, but I'm more confident when there's a different value anyway.

Reply:
It would be better to introduce a constant for this. Because then the name tells the reader what this value means and we can change it in a central location.